lib/internal/errors.js

```diff
@@ -703,6 +703,14 @@ E('ERR_INVALID_PERFORMANCE_MARK', 'The "%s" performance mark has not been set');
 E('ERR_INVALID_PROTOCOL', 'Protocol "%s" not supported. Expected "%s"');
 E('ERR_INVALID_REPL_EVAL_CONFIG',
   'Cannot specify both "breakEvalOnSigint" and "eval" for REPL');
+E('ERR_INVALID_SIGNATURE', (fn, args) => {
+  const formatted = lazyUtil().inspect(args, {
+    depth: 0,
+    breakLength: Infinity
+  }).replace(/^\[\s*/, '(').replace(/\s*\]$/, ')');
+  return `Invoking ${fn}() with invalid arguments: ${formatted}`;
```

Comment:
I feel like adding `combination` after `arguments` makes it clearer what it is about.